static/css/ubuntu.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-display: swap;
+	src:url('../fonts/Ubuntu-Regular.ttf') format('truetype');
+	font-weight: 100 700;
```

Comment:
Do we need 100 ?

Reply:
I guess not that's the value other fonts have in the codebase too. Could go 400 which is default for normal?